int64 timestamp
float32 temperature
float32 distance
float32 amplitude
